packages/vue/src/components/organisms/SfHeader/SfHeader.js

```diff
@@ -1,3 +1,56 @@
+import SfIcon from "../../atoms/SfIcon/SfIcon.vue";
+import SfImage from "../../atoms/SfImage/SfImage.vue";
+import SfSearchBar from "../../molecules/SfSearchBar/SfSearchBar.vue";
+
 export default {
-  name: "SfHeader"
+  name: "SfHeader",
+  components: {
+    SfIcon,
+    SfImage,
+    SfSearchBar
+  },
+  props: {
+    /**
+     * Header logo
+     */
+    logo: {
+      type: [String, Object],
+      default: ""
+    },
+    /**
+     * Header title
+     */
+    title: {
+      type: String,
+      default: ""
+    },
+    /**
+     * Header navigations
+     */
+    navigations: {
```

Comment:
this shouldn't be a prop (according to spec in issue) but a slot. By making it a prop we have no control over how clicks/hovers are handled for each menu element

Reply:
Changed in https://github.com/DivanteLtd/storefront-ui/pull/450/commits/c1385a3e3395189441fac8b7734bf2ea13bba3e4